DOCS-3785: Add module API key connection note for bot (#4171)
viamrobotics/docs: docs/dev/reference/apis/fleet.md

Once you have instantiated an `AppClient`, you can run the following [API methods](#api) against the `AppClient` object (named `fleet` in the examples).

{{% hiddencontent %}}
You cannot instantiate an `AppClient` from inside a module without authenticating using API keys.

> If your module requires an `AppClient`, you can use the module environment variables `VIAM_API_KEY` and `VIAM_API_KEY_ID` to authenticate.

Reviewer: Can you phrase this as a positive instead? "You must..."

> You cannot instantiate an `AppClient` from inside a module without authenticating using API keys.
> If your module requires an `AppClient`, you can use the module environment variables `VIAM_API_KEY` and `VIAM_API_KEY_ID` to authenticate.

Author: ```suggestion
To instantiate an `AppClient` from inside a module, you must authenticate using API keys.
You can use the module environment variables `VIAM_API_KEY` and `VIAM_API_KEY_ID` to access credentials.
```
